types/src/blob.rs

```diff
@@ -1,60 +1,44 @@
-use celestia_proto::tendermint::types as tmtypes;
-use serde::{Deserialize, Deserializer, Serialize};
+use serde::{Deserialize, Serialize};
+use tendermint_proto::v0_34::types::Blob as RawBlob;
+use tendermint_proto::Protobuf;
 
+use crate::nmt::Namespace;
+use crate::{Bytes, Error};
+
+// NOTE: We do not the `serce(try_from)` pattern for this type
+// becase JSON representation needs to have `commitment` field but
+// Protobuf definition doesn't.
+#[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct Blob {
-    blob: tmtypes::Blob,
-    commitment: Vec<u8>,
+    pub namespace: Namespace,
+    pub data: Bytes,
```

Comment:
can't we annotate those like with other protobuf types? this would allow us to remove the Bytes module
```
#[serde(with = "tendermint_proto::serializers::bytes::vec_base64string")]
```

Reply:
yes, i can do that